public/widget/discrepancy-note/Dn.js

```diff
@@ -582,13 +582,15 @@ Comment.prototype._renderHistory = function() {
     var over3 = this.notes.queries.concat( this.notes.logs ).length - 3;
     var $more = over3 > 0 ? $( '<tr><td colspan="4"><span class="over">+' + over3 + '</span>' +
         '<button class="btn-icon-only btn-more-history"><i class="icon"> </i></button></td></tr>' ) : $();
+    var $colGroup = this.notes.queries.concat( this.notes.logs ).length > 0 ? $( '<colgroup><col style="width: 31px;"><col style="width: auto;"></colgroup>' ) : $();
     this.$history.find( 'table' ).empty()
-        .append( '<thead><tr><td></td><td>' + historyText + '</td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
+        .append( $colGroup )
+        .append( '<thead><tr><td colspan="2"><strong>' + historyText + '</strong></td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
         .append( '<tbody>' +
             ( this.notes.queries.concat( this.notes.logs ).sort( this._datetimeDesc.bind( this ) ).map( function( item ) {
                     return that._getRows( item, true );
                 } )
-                .join( '' ) || '<tr><td colspan="2">' + emptyText + '</td><td></td><td></td></tr>' ) +
+                .join( '' ) || '<tr><td>' + emptyText + '</td><td></td><td></td><td></td></tr>' ) +
```

Comment:
I think it might be better to revert this line to the original. This way we have more space for emptyText (small screens, or verbose translation). We support a minimum of 320 px window size.

<img width="286" alt="screen shot 2017-06-08 at 3 33 23 pm" src="https://user-images.githubusercontent.com/627350/26951848-0579e0c4-4c60-11e7-8749-cb6df706e6c1.png">

with 1 `<td>` and colspan="2":
<img width="294" alt="screen shot 2017-06-08 at 3 33 53 pm" src="https://user-images.githubusercontent.com/627350/26951852-097a9ed4-4c60-11e7-9823-bcbdfdb541b6.png">